templates/static/normalized_os_version/data.csv

```diff
@@ -0,0 +1,51 @@
+os_name,os_version,normalized_os_name,normalized_os_version,release_date
```

Comment:
```suggestion
os_name,os_version_pattern,normalized_os_name,normalized_os_version,release_date
```

The fact that the os_version is a regex seems important and potentially unexpected. Somehow naming the column such that it's more obviously a regex could help discoverability. `os_version_regexp` would be another reasonable choice, especially since that matches REGEXP_CONTAINS.

Reply:
Yes `os_version_regexp` makes the most sense to me